CameraSourceBindings/Additions/AboutAdditions.txt

```diff
@@ -0,0 +1,48 @@
+﻿Additions allow you to add arbitrary C# to the generated classes
+before they are compiled.  This can be helpful for providing convenience
+methods or adding pure C# classes.
+
+== Adding Methods to Generated Classes ==
+
+Let's say the library being bound has a Rectangle class with a constructor
```

Comment:
Ist das richtiges Englisch? 
Evtl so? `Let's say the library which is bound, has a Rectangle....